modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeCatchAllJSPImportsCheck.java

```diff
@@ -17,11 +20,18 @@ protected String doProcess(
 			String fileName, String absolutePath, String content)
 		throws IOException {
 
-		if (!fileName.endsWith("UpgradeCatchAllCheck.jsp")) {
+		Matcher matcher = _pattern.matcher(fileName);
+
+		if (!matcher.find() ||
+			!absolutePath.contains("/upgrade/upgrade-catch-all-check")) {
+
 			return content;
 		}
 
 		return super.doProcess(fileName, absolutePath, content);
 	}
 
+	private static final Pattern _pattern = Pattern.compile(
+		"[A-Z]+_[0-9]+\\.jsp");
```

Comment:
If the pattern its to always at least LP for LPD, LPS, etc. 
The hotfix could start with that to make it even more closer to what we want to match, avoiding wrong match, its difficult to happen inside of our context but thinking in general development its a good practice try to protect yourself of those possibilities.

Reply:
@nicolas-sss I updated it to (LPD|LPS) (rather it just specifiy the two ticket types directly in this case instead of LP\W. I was worried that if we need to use tickets with new issueKey Types, but i guess if it comes to it then we can update it. Itll be better to be more specific as you mentioned for performance